File: 250324/학점 계산기/credit-calculator.py
n=int(input())
arr= list(map(float,input().split()))

total=sum(arr)


val=total/n
if val > 0:
    print(f"{val:.1f}")
    if val >= 4.0:
        print("Perfect")
    elif val >= 3.0:
        print("Good")
    else:
        print("Poor")

File: 250324/학점 계산기/credit-calculator.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;
    double arr[5];
    double total=0;
    // Please write your code here.
    for(int i=0; i<n; i++){
        cin >> arr[i];
    }
    for(int i=0; i<n; i++){
        total+=arr[i];
    }
    double val = total/n;
    cout << fixed;
    cout.precision(1);
    cout << val << endl;
    if(val>=4.0){
        cout << "Perfect";
    }
    else if(val >= 3.0){
        cout << "Good";
    
    }
    else{
        cout << "Poor";
    }
    return 0;
}